.circleci/config.yml

```diff
@@ -362,3 +370,5 @@ workflows:
       - just_simulate_sc_rehearsal_4
       - just_simulate_ink_respected_game_type
       # sepolia
+      - just_simulate_zora_002_fp_upgrade
```

Comment:
about start reviewing this, but noticed CI is failing with
```
[Revert] revert: Unallowed Storage access: 0xc26977310bC89DAee5823C2e2a73195E85382cC7
```

Reply:
This is the liveness guard, so seems ok to just add this to the allowed accesses array

Reply:
Weird, why did this not trigger when I simulate locally?

Reply:
Guessing you simulated as foundation but this will only show up for the SC